src/index.ts

```diff
@@ -26,13 +26,19 @@ function loader(this: interfaces.Webpack, contents: string) {
         return;
     }
 
+    // Load files generated by previous loaders in the same chain
+    const generatedFiles = getGeneratedFiles(this);
```

Comment:
Btw. I am happy with the option name `usePreviousLoaderGeneratedFiles`, but say the word, and I will change it.